lib/dependabot/utils/java_script/dependency_files.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+module Dependabot
+  module Utils
```

Comment:
We're getting rid of the `utils` namespace in the refactor (for this and all other languages). Can this like in the `FileUpdater` namespace, instead?